archive/archive_start_ingest/src/archive_start_ingest.py

```diff
@@ -35,39 +54,24 @@ def post_ingest_request(event, sns_client):
     logger.debug('ingest_request_id: %r', ingest_request_id)
 
     message = archive_bag_message(ingest_request_id, upload_url, callback_url)
-    logger.debug(f"sns-message: {message}")
+    logger.debug("sns-message: %r", message)
```

Comment:
y no f-string?

Reply:
premature optimisation ;)
Saves the cost of string building if it's never logged.